backend/src/bundles/recently-viewed/types/recently-viewed-templates-query-result.type.ts

```diff
@@ -0,0 +1,12 @@
+import { type RecentlyViewedModel } from '../recently-viewed.model';
+import { type TemplateDto } from './types';
+
+interface RecentlyViewedTemplatesQueryResult extends RecentlyViewedModel {
+    templates: TemplateDto;
+    resumes: {
```

Comment:
Consider moving to a separate type

Reply:
I've used it because there wasn't resume type. but I've already replace it with Resume type